action.yml

```diff
@@ -25,7 +25,7 @@ runs:
       cd "${GITHUB_WORKSPACE}" || exit 1
       TEMP_PATH="$(mktemp -d)"
       PATH="${TEMP_PATH}:$PATH"
-      curl -sfL https://raw.githubusercontent.com/Songmu/tagpr/v1.5.2/install.sh | sh -s -- -b "$TEMP_PATH" "${{ inputs.version }}" 2>&1
+      curl -sfL "https://raw.githubusercontent.com/Songmu/tagpr/${{ inputs.version }}/install.sh" | sh -s -- -b "$TEMP_PATH" "${{ inputs.version }}" 2>&1
```

Comment:
It is incorrect for the behavior to be dictated by `${{ inputs.version }}`, as this refers to the version of the tagpr binary, which can be arbitrarily set by the end user.

In GitHub Actions, users expect that the specified version of the action itself is what gets executed.

Therefore, the version of the action and the version of the installer should be expected to match.

Reply:
https://github.com/Songmu/tagpr/pull/204